<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommentsWidget</class>
 <widget class="QDockWidget" name="CommentsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Comments</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTreeView" name="commentsTreeView">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QuickFilterView" name="quickFilterView" native="true"/>
    </item>
   </layout>
  </widget>
  <action name="actionHorizontal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Horizontal</string>
   </property>
   <property name="toolTip">
    <string>Horizontal view</string>
   </property>
  </action>
  <action name="actionVertical">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vertical</string>
   </property>
   <property name="toolTip">
    <string>Vertical view</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QuickFilterView</class>
   <extends>QWidget</extends>
   <header>widgets/QuickFilterView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
